handlers/hormuud/handler_test.go

```diff
@@ -186,9 +186,6 @@ var tokenTestCases = []OutgoingTestCase{
 		MsgText: "Simple Message",
 		MsgURN:  "tel:+250788383383",
 		MockResponses: map[string][]*httpx.MockResponse{
-			"https://smsapi.hormuud.com/api/SendSMS": {
-				httpx.NewMockResponse(400, nil, []byte(`{"error": "invalid token"}`)),
-			},
```

Comment:
we shouldn't be making this call if fetching the token failed

Reply:
Yes